tests/basic/reboot_test.go

```diff
@@ -253,3 +253,77 @@ var _ = Describe("{ReallocateSharedMount}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{ClusterPxRestart}", func() {
+	var testrailID = 60042
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/60042
+	var runID int
+	loc, _ := time.LoadLocation("UTC")
+	t1 := time.Now().In(loc)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ClusterPxRestart", "Validate restart of PX in cluster", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+
+	stepLog := "has to restart Px cluster "
+	It(stepLog, func() {
+		dash.Info(stepLog)
```

Comment:
change to log.InfoD, this is new change where we pushing to log file and dashboard
log.Info only to log file